swing/src/net/sf/openrocket/gui/configdialog/RailButtonConfig.java

```diff
@@ -90,20 +97,30 @@ private JPanel buttonTab( final RailButton rbc ){
 			JSpinner offsetSpinner = new JSpinner(offsetModel.getSpinnerModel());
 			offsetSpinner.setEditor(new SpinnerEditor(offsetSpinner));
 			panel.add(offsetSpinner, "growx");
-			panel.add(new UnitSelector( offsetModel), "growx");
-			panel.add(new BasicSlider( offsetModel.getSliderModel(0, parentLength)), "w 100lp, wrap para");
+			panel.add(new UnitSelector(offsetModel), "growx");
+			panel.add(new BasicSlider(offsetModel.getSliderModel(
+					new DoubleModel(component.getParent(), "Length", -1.0, UnitGroup.UNITS_NONE),
+					new DoubleModel(component.getParent(), "Length"))),
+					"w 100lp, wrap para");
 			
 		}
 		
 		//// Material
-		panel.add( instanceablePanel(rbc), "cell 4 0, spany 3, wrap para");
+		/* TODO (wolsen) confirm this removal
```

Comment:
@wolsen 
Hey, rather than making a unilateral decision about this, I'd like to poll the list.

I understand your opinion. And I have the opposite opinion.
In addition, users don't *need* to use this panel.  Adding two separate rail buttons still works as expected.

Reply:
@teyrana 

Fair enough. I actually didn't intend to remove it as part of this PR. In my previous git commit, you can clearly see a comment with a TODO from me trying to find out if we should remove it. You've sent an email to the mailing list, so I'll respond in kind there.